os/arch/arm/src/amebad/amebad_flash.c

```diff
@@ -267,13 +267,19 @@ static ssize_t amebad_read(FAR struct mtd_dev_s *dev, off_t offset, size_t nbyte
  * Name: amebad_write
  ************************************************************************************/
 
-#if defined(CONFIG_MTD_BYTE_WRITE)
+#ifdef CONFIG_MTD_BYTE_WRITE
 static ssize_t amebad_write(FAR struct mtd_dev_s *dev, off_t offset, size_t nbytes, FAR const uint8_t *buffer)
 {
 	FAR struct amebad_dev_s *priv = (FAR struct mtd_dev_s *)dev;
+	size_t addr;
+	ssize_t result;
+	addr = CONFIG_AMEBAD_FLASH_BASE + offset;
+	result = amebad_flash_write(addr, buffer, nbytes);
+	return result < 0 ? result : nbytes;

```

Comment:
`nbytes` is unsigned type, `size_t`. Is it okay not to make type casting?

Reply:
1. it should return one of errno or written bytes. amebad_flash_write handles it.
2. ssize_t huge enough for TizenRT as a buffer. (I think enough for window or linux)
3. that is one of posix style(https://linux.die.net/man/3/write)